flutter_app/lib/widgets/drone_information_widget.dart

```diff
@@ -29,49 +29,53 @@ class DroneInformation extends StatelessWidget {
           color: Colors.black,
         ),
       ),
-      child: GridView.count(
-        crossAxisCount: 2,
+      child: SizedBox(
```

Comment:
The `SizedBox` widget is redundant, `Container` has the properties 'width' and 'height' so just add them there.